docs/user-guide/prereqs.md

```diff
@@ -71,13 +52,26 @@ $ sudo cp kubernetes/platforms/darwin/amd64/kubectl /usr/local/bin/kubectl
 $ sudo cp kubernetes/platforms/linux/amd64/kubectl /usr/local/bin/kubectl
 ```
 
-#### Configuring Kubectl
+You also need to ensure it's executable:
```

Comment:
I think the user needs this if he exports the path, so maybe move this block to after the instructions on exporting paths.